tests/integration/api/controllers/all-docs.spec.js

```diff
@@ -192,10 +192,10 @@ describe('all_docs handler', () => {
 
   it('filters offline users when requested with keys param', () => {
     const docs = [
-      { _id: 'allowed_contact', parent: { _id: 'fixture:offline'}, type: 'clinic' },
```

Comment:
I'm not sure what is adding these changes, there seem to be a lot of them and it makes reviewing this so much more cumbersome. I also like having spaces around curlies. Are these style changes (and IO'm referring to all changes in this PR that remove or add spaces around curlies) necessary in this already large PR?

Reply:
I'm so sorry about this. I had to reinstall my IDE because of an update going wrong, and now it seems to be changing the spaces! I will fix this for the next round of review.